lib/mixpanel-node.js

```diff
@@ -47,6 +49,8 @@ var create_client = function(token, config) {
             'verbose': metrics.config.verbose ? 1 : 0
         };
 
+        var protocol = metrics.config.protocol === 'http' ? http : https;
```

Comment:
Totally agree, changed in https://github.com/mixpanel/mixpanel-node/pull/96/commits/409c3e6c82f041ebf33dd2ad419c1914c20479f3